gapipy/models/base.py

```diff
@@ -126,7 +126,7 @@ def _set_resource_field(self, field, value):
             for v in value:
                 stub.append(self._model_cls(field)(v, stub=True))
         elif value:
-            stub = self._model_cls(field)(value, stub=True)
+            stub = self._model_cls(field)(value, self._client, stub=True)
         setattr(self, field, stub)
 
     def _set_resource_collection_field(self, field, value):
```

Comment:
Actually, the `Resource` class uses `self._client` in a few spots as well to fetch the full resource. So, it's less optional than originally thought. Hm.